vscode-lean4/src/docview.ts

```diff
@@ -83,7 +84,7 @@ export class DocViewProvider implements Disposable {
     }
 
     private async offerToOpenProjectDocumentation() {
```

Comment:
This appears to be dead code, please remove `offerToOpenProjectDocumentation`, thanks.